chart/values.schema.json

```
@@ -70,6 +70,332 @@
             "default": "2.1.3",
             "x-docsSection": "Common"
         },
+        "podSecurity": {
+            "description": "Set security contexts for certain containers",
+            "type": "object",
+            "x-docsSection": "Kubernetes",
+            "additionalProperties": false,
+            "properties": {
+                "default": {
+                    "description": "Default global security context.",
+                    "type": "object",
+                    "additionalProperties": false,
+                    "properties": {
+                        "securityContext": {
+                            "description": "Global Pod security context as defined in https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#podsecuritycontext-v1-core",
+                            "type": "object",
+                            "default": "See values.yaml",
```

```suggestion
                            "default": {"runAsUser": 50000, "fsGroup": 0, "runAsGroup": 0},
```

I think we should bring the actual defaults into the docs instead.

---

Hi @jedcunningham, the changes were applied as requested.